app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<body>
+    <% @users.each do |user| %>
+        <%= link_to "/users/#{user.id}" do %>
+        <%= render "partials/user_container", user: user %>
+        <% end %>
+    <% end %>
+</body>
```

Comment:
- [ ] kindly make sure your design match the given wire frame
your design :point_down: 
![Screenshot from 2023-04-21 17-06-58](https://user-images.githubusercontent.com/66097371/233683750-5f1dad3b-5519-48f8-9b72-a8f41f8707c0.png)
given wire frame :point_down: 
![Screenshot from 2023-04-21 17-06-41](https://user-images.githubusercontent.com/66097371/233683819-d0d30ed0-1584-4b17-afc4-f173ab651cbe.png)